### DISCUSSION TITLE
[Feature Request] Cycling Rich Prescence

### DISCUSSION THREAD

**Discussion by BirbEXE** [maximmax42/Discord-CustomRP#79] [Ideas]
hey, I saw a missing feature, and thought it'd be a cool addition!
A cycling presence would allow CustomRP to switch between prescence preset files every so often:
for instance, you could load up to 5 preset files which would rotate, in a set order, every so long

thanks for considering my suggestion, I appreciate it!
~birb

**Comment by maximmax42** [comment]
Yeah someone else also suggested a similar feature, but with animating fields separately. If the updates are gonna be like once a minute or slower - sure, that can work, but if like once in 10 seconds - it might result in some sort of timeout, so I don't know. I'm also not sure how to implement this UI-wise, can't think of an intuitive way to do it.

**Reply by MayhemBill** [reply]
You could have a drop down where users can select them? like *Frame 1* would open up just a single selections for status then *Frame 2* could select the next one. If I could actually make an example it would be easier to show I think.

**Reply by BirbEXE** [reply]
That's not a bad idea actually

**Reply by maximmax42** [reply]
I'm not sure how would it work with just dropdowns.

**Reply by MayhemBill** [reply]
Not drop downs, but a button that would open a new smaller window, you'd fill out the details, then hit "save" and the new window would go away

Or you could have a layout like it is now, but a button that says "next" and "previous" and you could cycle through the different frames?

**Reply by maximmax42** [reply]
Hmmmmmmmmmm.

**Reply by MayhemBill** [reply]
I feel like the buttons per frame would limit how many buttons you could have comfortably, but cycling through frames would be a bit of a hassle, so it's a pick whichever you like better kind of thing.
